apps/src/applab/appStorage.js

```diff
@@ -38,6 +38,10 @@ var handleGetKeyValue = function(onSuccess, onError) {
   if (this.readyState !== 4) {
```

Comment:
Ahh, didnt realize this was an outside of our code thing. Surprised constants for these don't exist already.